services/route/server.go

```diff
@@ -112,6 +112,6 @@ func computeRoute(ctx context.Context, pickup, dropoff string) *Route {
 	return &Route{
 		Pickup:  pickup,
 		Dropoff: dropoff,
-		ETA:     time.Duration(eta) * time.Hour,
+		ETA:     -1 * time.Duration(eta) * time.Hour,
```

Comment:
Note: Added to demonstrate test run and fail for the right reasons. Will revert before merge.